Controllers/Topic.controller.js

```diff
@@ -0,0 +1,9 @@
+const selectTopics = require("../Models/Topic.model.js");
+
+const getTopics = (req, res) => {
+  selectTopics().then((topics) => {
+    res.status(200).send(topics);
```

Comment:
The response is not in the correct the format specified on the ticket: {topics: [...]}

Reply:
Hi alex,
reading through the ticket , this is what i took away from it. i cant see a  mention of a topics field. I thought we should return an array of topic objects?